test/release/examples/primers/arrays.chpl

```diff
@@ -61,176 +72,229 @@ writeln("After assigning its interior values, A is: ", A);
 writeln();
 
 //
-// Just like with array indexing, either square brackets or parentheses
-// can be used for array slicing:
+// As with array indexing, either square brackets or parentheses can
+// be used for array slicing:
 //
 
 writeln("A(2..4) is: ", A(2..4), "\n");
 
 //
-// Note: further information on slicing can be found in the
-// :ref:`Slices Primer <primers-slices>`
+// Further information on slicing can be found in the :ref:`Slices
+// Primer <primers-slices>`
 //
 
+// Multidimensional Arrays
+// -----------------------
+
 //
-// Arrays can be multidimensional as well:
+// Arrays can be multidimensional as well.  For example, the following
+// declaration creates a 2D ``n`` x ``n`` array of ``real``
+// values.
 //
 
 var B: [1..n, 1..n] real;
 
 forall (i,j) in {1..n, 1..n} do
-  B(i,j) = i + j/10.0;
+  B[i,j] = i + j/10.0;
 
 writeln("Initially, B is:\n", B, "\n");
 
+// Loops over Arrays
+// -----------------
+
 //
-// An array's elements can be iterated over in a for or forall loop,
-// which causes the index variable to store references to an array's
-// elements:
+// An array's elements can be iterated over using Chapel's standard
+// loop forms like ``for``, ``foreach``, or ``forall``.  These cause
+// the index variable to refer to an array element in each iteration.
+// For example, the following loop increments each of ``B``'s elements
+// by 1, in parallel:
 //
 
 forall b in B do
   b += 1;
 
 writeln("After incrementing B's elements, B is:\n", B, "\n");
 
+// While the following loop negates ``A``'s elements:
+
+
```

Comment:
Missing a code example here?

Reply:
D'oh!  No, I just meant to remove this comment as well.  Thanks for the catch.